phpunit.xml.dist

```diff
@@ -1,17 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="./vendor/phpunit/phpunit/phpunit.xsd" bootstrap="vendor/autoload.php">
-  <testsuites>
-    <testsuite name="Koriym.Attribute test suite">
-      <directory>tests</directory>
-      <directory phpVersion="8.0.0" phpVersionOperator=">=">tests-php8</directory>
-    </testsuite>
```

Comment:
It seems this was skipping all tests on PHP 7.3 and 7.4.

This PR fixes it  :+1: